scripts/src/java/org/oppia/android/scripts/proto/BUILD.bazel

```diff
@@ -62,3 +62,15 @@ java_lite_proto_library(
     visibility = ["//scripts:oppia_script_library_visibility"],
     deps = [":maven_dependencies_proto"],
 )
+
+proto_library(
+    name = "accessibility_label_exemptions_proto",
+    srcs = ["script_exemptions.proto"],
```

Comment:
I suggest renaming the existing file & creating a new one--we shouldn't be reusing the same .proto file in multiple libraries. That leads to duplicate build graph work, and potentially duplicate symbols/single version violations in the graph which can actually cause runtime issues in the worst cases.

Reply:
Done.